src/components/content.js

```diff
@@ -148,24 +148,7 @@ class Content extends React.Component {
 
   getPoint(element, offset) {
```

Comment:
Can we remove this `#getPoint` method entirely now, and just use the new utility function that you created inline in each of the places?